velox/connectors/hive/HiveConfig.h

```diff
@@ -49,6 +49,10 @@ class HiveConfig {
   static constexpr const char* kMaxPartitionsPerWritersSession =
       "max_partitions_per_writers";
 
+  /// Maximum number of bucketed count.
+  static constexpr const char* kMaxBucketCount = "max-bucket-count";
```

Comment:
This should be `hive.max-bucket-count`. Same with `max-partitions-per-writers`. Can you fix both?

Reply:
Also `kInsertExistingPartitionsBehavior` need the `hive.` prefix.